.env

```diff
@@ -25,6 +25,7 @@ MODELS=`[
     "datasetName": "OpenAssistant/oasst1",
     "description": "A good alternative to ChatGPT",
     "websiteUrl": "https://open-assistant.io",
+    # "endpoints": [{"url": "http://127.0.0.1:8080/generate_stream"}],
```

Comment:
Having the endpoint commented out causes an internal server error with:
 
```Internal server error: Unexpected token # in JSON at position 214```

Maybe we could remove it from the `.env` but expand the readme a bit ?

Basically the behavior is that without an `endpoints` array defined, it will try to use the hosted hugging face API inference endpoint. (see [here](https://github.com/huggingface/chat-ui/blob/df5a2eb42b37a540e63674e47358ef55934ae38f/src/lib/server/modelEndpoint.ts#LL13C3-

If it's defined we use those endpoints instead!

Reply:
I have removed the requested information from .env and and updated the readme. Let me know if you have any other concerns or feedback.